tests/unit/datastore/test_datastore.py

```diff
@@ -35,7 +35,7 @@ class TestRecord(DatastoreRecord):
 def test_datastore_describe():
     temp_path = tempfile.mkdtemp()
     storage = datastore.Datastore(temp_path)
-    assert storage.LMDB_MAP_SIZE == 1_000_000_000_000
+    assert storage.LMDB_MAP_SIZE == 10_000_000
```

Comment:
will have to change this check based on the resolution of https://github.com/nucypher/nucypher/pull/1741#discussion_r480524635